src/form-fields/text-field.js

```diff
@@ -2,15 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import TextField from 'material-ui/TextField';
 
+const helpTextStyle = {
+    fontSize: '12px',
+    color: 'rgba(0, 0, 0, 0.3)',
+    margin: 0,
+    marginTop: -4,
```

Comment:
Very minor - To me it would seem a little cleaner to make this a single CSS property rather than relying on the property order - `margin: "-4px 0 0"` - otherwise I'd be afraid someone would look at this style block and stop looking at `margin: 0`, or they would get re-ordered in a refactor?